### PR TITLE
nit: Remove authenticators from state

### PR DESCRIPTION
**Pull request by schew2381**
Remove `authenticators` from state because we only set it once

### CODE REVIEW

File: static/app/components/superuserStaffAccessForm.tsx [hunk 1]
```diff
@@ -60,16 +59,17 @@ class SuperuserStaffAccessForm extends Component<Props, State> {
     }
 
     const authenticators = await this.getAuthenticators();
-    this.setState({authenticators: authenticators});
+    this.authenticators = authenticators;
```
Comment (nhsiehgit) [inline static/app/components/superuserStaffAccessForm.tsx:62]: :squint:

Ok so - we _shouldn't_ be utilizing non lifecycle state variables like this i don't think 🤔

**Review by nhsiehgit — COMMENTED** [review]
mmmm i'm not quite sure about this change 🤔

File: static/app/components/superuserStaffAccessForm.tsx [hunk 2]
```diff
@@ -39,8 +38,8 @@ class SuperuserStaffAccessForm extends Component<Props, State> {
   constructor(props) {
     super(props);
     this.authUrl = this.props.hasStaff ? '/staff-auth/' : '/auth/';
+    this.authenticators = [];
```
Comment (saponifi3d) [inline static/app/components/superuserStaffAccessForm.tsx:41]:
```suggestion
```
I'd recommend removing this and define it inline where you have the types , like this:
```
authenticators: Authenticator[] = [];
```
Which will do the same as what you have here.

**Review by saponifi3d — APPROVED** [review]
:shipit: just had the nit.

**Comment by cathteng** [comment]
shoot i reverted this by accident, i can put it back up

**Pull request merged**